package.json

```diff
@@ -21,6 +21,7 @@
     "eslint-plugin-fetch-options": "^0.0.5",
     "eslint-plugin-html": "^7.1.0",
     "eslint-plugin-import": "^2.29.1",
+    "eslint-plugin-jasmine": "^4.1.3",
```

Comment:
Looking at https://github.com/tlvince/eslint-plugin-jasmine it hasn't been updated for over two years, but I'm assuming that there's no more "maintained" alternative available?

Reply:
Yes I noticed that too but I can imagine that it's stable enough to not require any updates.
I didn't find any alternatives.